docs/src/reference/remote_inputs.md

```diff
@@ -71,7 +70,6 @@ This means that the full GenBank metadata and sequences are typically updated a
 | Full GenBank data    | metadata  | https://data.nextstrain.org/files/ncov/open/metadata.tsv.gz            |
 |                      | sequences | https://data.nextstrain.org/files/ncov/open/sequences.fasta.xz         |
 |                      | aligned   | https://data.nextstrain.org/files/ncov/open/aligned.fasta.xz           |
-|                      | filtered  | https://data.nextstrain.org/files/ncov/open/filtered.fasta.xz          |
```

Comment:
Reminder to us to remove this file after the first successful run (and the corresponding GISAID file as well)